tools/validators/instance_validator/validate/generate_universe.py

```diff
@@ -38,7 +38,7 @@ def BuildUniverse():
     print('ERROR: ontology validator or ontology have changed locations')
     return None
 
-  yaml_files = external_file_lib._RecursiveDirWalk(path.join(
+  yaml_files = external_file_lib.RecursiveDirWalkPublic(path.join(
```

Comment:
remove public

Reply:
It should pass the caps first method, see here: https://github.com/google/digitalbuildings/blob/master/tools/rdf_generator/rdfformat/generator/yaml_handler.py

Reply:
Done